currency_fetcher_cr.py

```diff
@@ -0,0 +1,61 @@
+import json
+import urllib.request
+from datetime import datetime
+from time import mktime, strptime
+from urllib.error import URLError
```

Comment:
Changed in commit [#5](https://github.com/ZlatyChlapec/MoneyHoney/commit/7f8b23b635ed1b7ce666cef2a04822397758a22a#diff-098d0331f00cb6dd29209b19ba26f515)